delivery_location/tests/test_deliverylocation.py

```diff
@@ -0,0 +1,96 @@
+import pytest
+from delivery_location.models import Supplier, DeliveryLocation
+from django.core.exceptions import ValidationError
+import datetime
+
+test_data = [
+    ('taxas', 'firstname', 'lastname', 'password', 'supplier', 'business name'),
+    ('ed', 'edd', 'eddy', 'jawbreaker101', 'supplier', 'jawbreaker inc'),
+]
```

Comment:
If it's a const then it needs to be in uppercase, but it's better to have that as a fixture. This way, if one of the tests accidentally modifies it, you get a fresh copy on the next test.

Reply:
fixed